charts/newrelic-infrastructure-beta/templates/kubelet/integrations-configmap.yaml

```diff
@@ -0,0 +1,14 @@
+{{ if .Values.integrations_config }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: {{ .Release.Namespace }}
+  labels: {{ include "newrelic.labels" . | nindent 4 }}
+  name: {{ template "newrelic.fullname" . }}-integrations-cfg
+data:
+{{ range .Values.integrations_config -}}
+{{ .name | indent 2 }}: |
+    ---
+{{ toYaml .data | indent  4 }}
```

Comment:
```suggestion
  {{- range $key, $value := .Values.integrations_config -}}
  {{ $key }}: |
    {{- toYaml $value | indent  4 }}
```

Reply:
It _should_ be like this, but the `integrations_config` thing has a very weird format in the values:
```yaml
# integrations_config:
#   - name: nri-redis.yaml
#     data:
#       discovery:
```
So we need to use that weird range.

Not sure if it's worth it changing it now, and adding it to the compat layer. WDYT?

Reply:
Compatibility layer. Another NTH.